### PR TITLE
fix broken link

### PR DESCRIPTION
**Pull request by saintmalik**
title: Fix broken link in docs/developer/guide/drupal/
  
  description: Easy navigation and easy reading.

### CODE REVIEW

File: content/guide/drupal/index.md [hunk 1]
```diff
@@ -42,7 +42,7 @@ You can install LoginRadius  Drupal Plugin [manually](#install-manually) or via
 
 ### Install via Web Interface
 
-1. Download the <a href="https://github.com/LoginRadius/drupal-identity-module/blob/master/drupal-8/package/customer_identity_and_access_management.tar" target="_blank">Drupal v8.x</a> or <a href="https://github.com/LoginRadius/drupal-identity-module/blob/master/drupal-9/package/customer_identity_and_access_management.tar" target="_blank">Drupal v9.x</a> Customer Identity module.
+1. Download the <a href="(https://github.com/LoginRadius/drupal-identity-module/blob/master/drupal-8/package/customer_identity_and_access_management.tar" target="_blank">Drupal v8.x</a> or <a href="https://github.com/LoginRadius/drupal-identity-module/blob/master/drupal-9/package/customer_identity_and_access_management.tar" target="_blank">Drupal v9.x</a> Customer Identity module.
```
Comment (pareek-narendra) [inline content/guide/drupal/index.md:45]: @saintmalik 
Thank you for your contribution.
You have made the change at the wrong place, it is working fine.

You can fix the broken link under the "**Install Manually**" heading (just need to remove the  "(" **start bracket** )

Comment (pareek-narendra) [reply content/guide/drupal/index.md:45]: Also, raise the PR for dev branch not for main

**Review by pareek-narendra — CHANGES_REQUESTED** [review]
Request changes

File: content/guide/drupal/index.md (hunk 1 above)
Comment (saintmalik) [reply content/guide/drupal/index.md:45]: okay thanks, will do that

Comment (pareek-narendra) [reply content/guide/drupal/index.md:45]: @saintmalik please let me know if you are fixing this.

Comment (saintmalik) [reply content/guide/drupal/index.md:45]: i will be fixing it, on it

**Comment by saintmalik** [comment]
the isssue have been fixed with this pull request #198 , am closing this

**Pull request closed without merging**